app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -104,8 +107,47 @@ private val SKILL_ID_LIST = listOf(FRACTIONS_SKILL_ID_0)
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(application = QuestionPlayerActivityTest.TestApplication::class, qualifiers = "port-xxhdpi")
 class QuestionPlayerActivityTest {
-  // TODO(#503): add tests for QuestionPlayerActivity (use StateFragmentTest for a reference).
+  // TODO(#503): Add more test-cases
```

Comment:
why updating this TODO?, I guess until we get to the end of this PR either we can keep this for reference or we can remove this line.

Reply:
I will remove the TODO at the end of this PR.

Reply:
Keeping this open as a reminder to remove the TODO, else you can remove it now.